lib/thor.rb

```diff
@@ -152,6 +152,7 @@ def method_options(options = nil)
     # :type     - The type of the argument, can be :string, :hash, :array, :numeric or :boolean.
     # :banner   - String to show on usage notes.
     # :hide     - If you want to hide this option from the help.
+    # :inverse  - Set to false if you want to suppress --no-<option> to be generated for booleans.
```

Comment:
Let's change this to reflect what it's doing - so something like "Takes a string to customize the reverse option generated for booleans, or the value false to suppress it entirely."